src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_BinaryOperator.cs

```diff
@@ -2087,6 +2095,57 @@ private BoundExpression MakeNullCheck(SyntaxNode syntax, BoundExpression rewritt
                     type);
         }
 
+        private BoundExpression RewriteBuiltInNativeShiftOperation(
+            BoundBinaryOperator? oldNode,
+            BinaryOperatorKind operatorKind,
+            BoundExpression loweredLeft,
+            BoundExpression loweredRight,
+            TypeSymbol type)
+        {
+            // For the predefined operators, the number of bits to shift in `x << count`, `x >> count` or `x >>> count` is computed as follows:
+            // [...]
+            // - When the type of `x` is `nint` or `nuint`, the shift count is given by
+            //   the low-order five bits of `count` on a 32 bits platform, or
```

Comment:
>32 bits platform [](http://example.com/codeflow?start=57&length=16)

Consider using "... bit platform" (singular "bit") throughout this comment to match the spec.